mon-pix/tests/helpers/setup-integration.js

```diff
@@ -0,0 +1,7 @@
+import setupIntl from './setup-intl';
+import { setupRenderingTest } from 'ember-mocha';
+
+export default function setupIntegration() {
```

Comment:
Je trouve que ce nom fait perdre la notion qu'un test d'intégration front / ember est un test de rendering. Peut-être que cette suggestion pourrait aider à garder l'intention et aussi rappeler qu'il y a `setupRenderingTest` quand on n'a pas besoin de `setupIntl` qui peut (à vérifier éventuellement) avoir un impact significatif sur l'exécution de nos tests d'intégration 

```suggestion
export default function setupIntlRenderingTest() {
```

Reply:
Sur le renommage en setupIntlRenderingTest: ok pour moi, je le laisserais bien par contre sur tous les tests présents, car à terme tout va être traduit et le switch de lange sera partout.